File: festival-web/src/main/java/cn/bdqfork/web/route/annotation/Produces.java
package cn.bdqfork.web.route.annotation;

import java.lang.annotation.*;

/**
 * 指定路由的produces
 *
 * @author bdq
 * @since 2020/2/12
 */
@Documented
@Retention(RetentionPolicy.RUNTIME)
@Target(ElementType.METHOD)
public @interface Produces {
    String[] value();
}

File: festival-kotlin-web/src/main/kotlin/cn/bdqfork/kotlin/web/route/annotation/Produces.kt
package cn.bdqfork.web.route.annotation

/**
 * 指定路由的produces
 *
 * @author bdq
 * @since 2020/2/12
 */
@MustBeDocumented
@kotlin.annotation.Retention(AnnotationRetention.RUNTIME)
@Target(AnnotationTarget.FUNCTION)
annotation class Produces(vararg val value: String)